### PR TITLE
[PM] Wakelock feature for PGP in mwc service

### PR DESCRIPTION
**Pull request by shwethabolumbu**
To inhibit the host suspend activity when
SCREEN_BRIGHT_WAKE_LOCK is held. The service is
started along with the mwc service during Host Boot.

Tracked-On: OAM-100468
Signed-off-by: Shwetha B <shwetha.b@intel.com>

### CODE REVIEW

**Review by kaushlen — APPROVED** [review]

**Review by shuangwan01 — APPROVED** [review]
LGTM

**Review by kaushlen — APPROVED** [review]

**Comment by xihua-chen** [comment]
If no host tool of python3-pip & sleep-inhibitor, what will happen?

File: host/src/pack/dpkg/etc/rc.mwc [hunk 1]
```diff
@@ -12,6 +12,23 @@ do
     chmod 666 /dev/shm/looking-glass$i
 done
 
+echo "Set sleep-inhibitor if not already done..."
+if [ ! -d /usr/local/share/sleep-inhibitor/plugins/is-wakelock-active ]
+then
+       cp /opt/cfc/mwc/bin/is-wakelock-active /usr/local/share/sleep-inhibitor/plugins/
```
Comment (renchenglei) [inline host/src/pack/dpkg/etc/rc.mwc:18]: Can we follow the same format as above?

File: host/src/pack/dpkg/etc/rc.mwc [hunk 2]
```diff
@@ -12,6 +12,23 @@ do
...
 done
 
+echo "Set sleep-inhibitor if not already done..."
+if [ ! -d /usr/local/share/sleep-inhibitor/plugins/is-wakelock-active ]
+then
+    cp /opt/cfc/mwc/bin/is-wakelock-active /usr/local/share/sleep-inhibitor/plugins/
+fi
+if [ ! -d /etc/sleep-inhibitor.conf ]
+then
+   cp /opt/cfc/mwc/bin/sleep-inhibitor.conf /etc/
```
Comment (renchenglei) [inline host/src/pack/dpkg/etc/rc.mwc:22]: maybe we need 4 space?

File: host/src/pack/dpkg/etc/rc.mwc [hunk 3]
```diff
@@ -12,6 +12,23 @@ do
...
+fi
+if [! -d /etc/systemd/system/sleep-inhibitor.service ]
+then
+   cp /etc/systemd/user/sleep-inhibitor.service /etc/systemd/system/
```
Comment (renchenglei) [inline host/src/pack/dpkg/etc/rc.mwc:26]: add here?

File: host/src/pack/dpkg/opt/cfc/mwc/bin/is-wakelock-active [hunk 4]
```diff
@@ -0,0 +1,17 @@
+#! /bin/sh
+if [ "$(systemctl --user is-active civ | grep -w 'active')" ] && [ "$(pidof mwc_launcher)" ] && [ "$(pidof LG_B1_Client)" ]
+then
+	if adb get-state 1>/dev/null 2>&1
```
Comment (renchenglei) [inline host/src/pack/dpkg/opt/cfc/mwc/bin/is-wakelock-active:4]: add here?

File: host/src/pack/dpkg/opt/cfc/mwc/bin/is-wakelock-active [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+	then
+		state=$(adb shell dumpsys power | grep -oE 'SCREEN_BRIGHT_WAKE_LOCK')
```
Comment (renchenglei) [inline host/src/pack/dpkg/opt/cfc/mwc/bin/is-wakelock-active:6]: add here?

File: host/src/pack/dpkg/opt/cfc/mwc/bin/is-wakelock-active [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+		if echo "$state" | grep 'SCREEN_BRIGHT_WAKE_LOCK'; then
+			exit 254
```
Comment (renchenglei) [inline host/src/pack/dpkg/opt/cfc/mwc/bin/is-wakelock-active:8]: add here?

File: host/src/pack/dpkg/etc/rc.mwc (hunk 2 above)
Comment (shwethabolumbu) [reply host/src/pack/dpkg/etc/rc.mwc:22]: This should be fine. It is same as the above one

File: host/src/pack/dpkg/etc/rc.mwc (hunk 3 above)
Comment (shwethabolumbu) [reply host/src/pack/dpkg/etc/rc.mwc:26]: Same as the above one

File: host/src/pack/dpkg/etc/rc.mwc [hunk 7]
```diff
@@ -12,6 +12,23 @@ do
...
+then
+    cp /opt/cfc/mwc/bin/sleep-inhibitor.conf /etc/
+fi
+if [! -d /etc/systemd/system/sleep-inhibitor.service ]
+then
+    cp /etc/systemd/user/sleep-inhibitor.service /etc/systemd/system/
+fi
+
+systemctl daemon-reload
+systemctl start sleep-inhibitor
```
Comment (YadongQi) [inline host/src/pack/dpkg/etc/rc.mwc:30]: put conf/service file directly to the correct path, these copy steps are unecessary

**Pull request closed without merging**